src/main/kotlin/no/nav/syfo/dialogmote/api/domain/DialogmoteDTO.kt

```diff
@@ -17,4 +17,5 @@ data class DialogmoteDTO(
     val tid: LocalDateTime,
     val videoLink: String?,
     val referat: ReferatDTO?,
```

Comment:
Beholder referat her for å ikke knekke Modia, men bør fjernes når Modia har blitt tilpasset til å bruke referatList i stedet for.